.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "build": {
+        "dockerfile": "Dockerfile",
+        "args": {
+            "docker_image_tag_hash": "b5b06ad736cc8f2df227dabf8ff1da1a2f538939"
```

Comment:
An idea could be to also have a `22.04-b5b....939`?

Reply:
you mean to avoid the long hash?

Reply:
I do not know how you generate this hash. It looks to me that you use one unique image for all your builds?
In geos, we "play" with the repository name (like `ubuntu-22-gcc10...`) and the hash is unique and matches versions of the TPLs. It depends on what you want to do for Shiva. Still, it's no big deal to make this evolve in the future.

Reply:
the hash is from the [docker_base_images](https://github.com/GEOS-DEV/docker_base_images) repository. I would like to standardize the images we use for testing across the project repos.